### PR TITLE
Run fast pass on ims #507

### LINKED ISSUE #507: run fast pass on on pages on the website
- install the extension below 
- https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni?hl=en-GB&utm_source=ext_sidebar
- Run fast pass and fix all of the errors

### PR DESCRIPTION
**Pull request by MatteoGuarnaccia5**
## Description

Ran fastPass on IMS, see #507 for more details

## Testing instructions

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage

## Agile board tracking

closes #507

### CODE REVIEW

**Comment by MatteoGuarnaccia5** [comment]
waiting on #525 and #542 to continue progress

**Comment by joshuadkitenge** [comment]
Do this on Dev instance

File: src/catalogue/items/catalogueItemsDialog.component.tsx [hunk 1]
```diff
@@ -977,6 +977,7 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
                           sx={{ display: 'flex', alignItems: 'center' }}
                         >
                           <Tooltip
+                            aria-label={`${property.name} tooltip`}
```
Comment (joshuadkitenge) [inline src/catalogue/items/catalogueItemsDialog.component.tsx:980]:
```suggestion
                            aria-label={`${property.name} details`}
```

File: src/view/breadcrumbs.component.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const Breadcrumbs = (props: BreadcrumbsProps) => {
         <HomeIcon />
       </IconButton>
       <StyledBreadcrumbs
-        separator={<NavigateNext fontSize="small" />}
+        separator={<NavigateNext fontSize="small" aria-label="navigate next" />}
```
Comment (joshuadkitenge) [inline src/view/breadcrumbs.component.tsx:52]:
```suggestion
        separator={<NavigateNext fontSize="small" />}
```

Is this still needed ???

**Review by joshuadkitenge — DISMISSED** [review]

**Review by joelvdavies — CHANGES_REQUESTED** [review]
Just one minor thing, looks good otherwise.

**Review by joshuadkitenge — APPROVED** [review]

**Review by joelvdavies — APPROVED** [review]

**Pull request merged**